### DISCUSSION TITLE
Can I Join the team?

### DISCUSSION THREAD

**Discussion by orkunergun** [ROM-builders/temporary#10591] [General]
My builds :
Evox - https://cirrus-ci.com/build/5635217607098368
Cherish OFFİCİAL - https://cirrus-ci.com/build/6525580903448576
Cherish Merlinx - https://cirrus-ci.com/build/4539243488870400
Yaap - https://cirrus-ci.com/build/4538994917638144
 Xdroid - https://cirrus-ci.com/build/6475986949636096
 Arrow - https://cirrus-ci.com/build/5332538678837248
Awaken - https://cirrus-ci.com/build/5980191829786624
Awaken Merlinx - https://cirrus-ci.com/build/6041897054502912
Cipher - https://cirrus-ci.com/build/5908461580976128

Device Trees : https://github.com/r9lab
Telegram username : @orkunergun
I'm maintaining 2 roms now (cherish and crystal)
I will switch gitlab for my trees so you can ignore commit names I will change this.

**Comment by Apon77** [comment]
Check #learn in telegram group

**Reply by orkunergun** [reply]
Already and still learning git

**Reply by orkunergun** [reply]
I'm just tired of the approval event and I need to do cherish monthly.

**Reply by orkunergun** [reply]
And the team is still extending the time :/
